compiler/luci/pass/src/QuantizeWithMinMaxPass.cpp

```diff
@@ -580,7 +560,7 @@ struct QuantizeActivation final : public luci::CircleNodeMutableVisitor<bool>
         continue;
 
       // Check if this is bool type (bool type is not quantized)
-      if (bool_type(circle_node))
+      if (circle_node->dtype() == loco::DataType::BOOL)
```

Comment:
this change doesn't seem to be related with `LogicalOr`. It would be better to post in anothr PR.

Reply:
I'll rebase this PR after #6498 is merged.